helpers/otg/otg.py

```diff
@@ -264,6 +264,42 @@ def get_bgpv4_metrics(self):
         finally:
             self.timer("get_bgpv4_metrics", start)
 
+    def get_isis_metrics(self):
+        start = datetime.datetime.now()
+        try:
+            log.info("Getting isis metrics ...")
+            req = self.api.metrics_request()
+            req.isis.router_names = []
+
+            metrics = self.api.get_metrics(req).isis_metrics
+
+            tb = table.Table(
+                "ISIS Metrics",
+                [
+                    "Name",
+                    "L1 Sessions Up",
+                    "L2 Sessions UP",
+                    "L1 Database Size",
+                    "L2 Database Size",
+                ],
```

Comment:
add more spacing in columns - current one doesn't look ok

Reply:
updated